src/http/any-catchall/index.js

```diff
@@ -1,14 +1,11 @@
-// eslint-disable-next-line
```

Comment:
Can we just re-name these files to .mjs and then we can get rid of the package.json -> type: module files?

Reply:
That's a good point. I can't remember why that approach didn't work at first, but I will give it another shot and hopefully get rid of the stray package files